api/src/test/java/org/openmrs/module/sync/SyncPrivilegeTest.java

```diff
@@ -60,7 +61,8 @@ public void runOnParent() throws Exception {
 			}
 		});
 	}
-	
+
+	@Disabled("I am disabling this test, as this should not be allowed. privilege is unique, so no 2 privileges can exist with the same privilege and different uuids")
```

Comment:
See comment in the disabled annotation.  This was failing, but I can't really see how it should ever work?